e2e/cypress/integration/account_settings/display/account_settings_position_spec.js

```diff
@@ -0,0 +1,93 @@
+// ***************************************************************
+
+// Group: @account_setting
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Account Settings > General > Position', () => {
+    let testTeam;
+    let testUser;
+    const position = 'Master hacker';
+
+    beforeEach(() => {
+        cy.apiAdminLogin();
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+
+            cy.apiLogin(testUser);
+
+            // # Visit town square
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-T2063 Position', () => {
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on Account settings menu item
+        cy.findByText('Account Settings').should('be.visible').click();
+
+        // # Fill-in the position field
+        cy.findByText('Position').should('be.visible').click();
+        cy.get('#position').type(position);
+        cy.get('#saveSetting').click();
+        cy.get('#position').should('not.be.visible');
+
+        // # Exit the modal
+        cy.get('body').type('{esc}', {force: true});
+
+        // # Post message in the main channel
+        cy.postMessage('hello from master hacker');
+
+        // # Click on the profile image
+        cy.get('.profile-icon > img').as('profileIconForPopover').click();
+
+        // # Verify that the popover is visible and contains position
+        cy.contains('#user-profile-popover', 'Master hacker').should('be.visible');
+    });
+
+    it('MM-T2064 Position / 128 characters', () => {
+        const longPosition = 'Master Hacker II'.repeat(8);
+
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').should('be.visible').click();
+
+        // # Click on Account settings menu item
+        cy.findByText('Account Settings').should('be.visible').click();
+
+        // # Fill-in the position field with a value of 128 characters
+        cy.findByText('Position').should('be.visible').click();
+        cy.get('#position').type(longPosition);
+        cy.get('#saveSetting').click();
+        cy.get('#position').should('not.be.visible');
+
+        cy.get('#position').invoke('val').then((val) => {
+            // # Verify that the input value is 128 characters
+            expect(val.length).to.equal(128);
+        });
+
+        // # Try to edit the field with maximum characters
+        cy.findByText('Position').click();
+        cy.wait(TIMEOUTS.ONE_SEC);
+
+        cy.get('#position').should('be.visible').focus().type('random');
+        cy.get('#position').invoke('val').then((val) => {
+            // # Verify that the position hasn't changed
+            expect(val).to.equal(longPosition);
+        });
+
+        // # Save position
+        cy.get('#saveSetting').click();
+
+        // # Exit the modal
+        cy.get('body').type('{esc}', {force: true});
```

Comment:
Can't see where "After 4. The full position saves". Could you please take a look again? Thanks

Reply:
it's in line 88, where I am able to save the setting and before that even checked that it equals the position I specified.

Reply:
Ah I see. I thought the verification is to be done at `#positionDesc` but it's actually not clear in the test case so the approach here makes sense too. Thanks for checking again.
<img width="622" alt="Screen Shot 2020-12-09 at 8 37 32 PM" src="https://user-images.githubusercontent.com/5334504/101631300-198e6a80-3a5f-11eb-9f98-7c5b97a18611.png">